src/main/java/bq_standard/importers/hqm/converters/rewards/HQMRewardCommand.java

```diff
@@ -0,0 +1,28 @@
+package bq_standard.importers.hqm.converters.rewards;
+
+import betterquesting.api.questing.rewards.IReward;
+import bq_standard.rewards.RewardCommand;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HQMRewardCommand {
+    public IReward[] convertReward(JsonElement json) {
+        if (!(json instanceof JsonArray)) return null;
+
+        List<IReward> rList = new ArrayList<>();
+
+        for (JsonElement je : json.getAsJsonArray()) {
+            if (!(je instanceof JsonPrimitive)) continue;
+            RewardCommand reward = new RewardCommand();
+            reward.command = je.getAsString();
+            reward.viaPlayer = true;
```

Comment:
Not sure if it matters, or what it does, but the original code had another line here:
https://github.com/GTNewHorizons/BetterQuesting/blob/188ca9df26c3d4205a8d07b2dc94b409b5c3a1b6/src/main/java/bq_standard/importers/hqm/converters/rewards/HQMRewardCommand.java#L22

I guess the most likely explanation is that that line was removed at some point after the original backport was done?